roles/os_hardening/tasks/sysctl.yml

```diff
@@ -48,6 +48,17 @@
         sysctl_config: '{{ sysctl_config | combine(sysctl_overwrite) }}'
```

Comment:
I think this should be the last merge task, only then we are able to use `sysctl_overwrite` to change values in `sysctl_custom_config` or `sysctl_rhel_config`